enterprise/app/repo/repo.tsx

```diff
@@ -250,15 +250,20 @@ export default class RepoComponent extends React.Component<RepoComponentProps, R
     r.template = this.state.template;
     r.templateDirectory = this.state.templateDirectory;
     r.destinationDirectory = this.state.destinationDirectory;
+
+    if (selectedInstallation) {
+      r.owner = selectedInstallation.login;
+      r.installationId = selectedInstallation.id;
+    }
+
     if (update) {
       r.skipRepo = true;
       r.skipLink = true;
       r.templateVariables = Object.fromEntries(this.state.vars.entries());
     }
 
     if (selectedInstallation?.targetType == "Organization") {
```

Comment:
Does it make sense to just pass up installation target type directly instead of converting it to a bool?

Reply:
Good idea, done!